Formula/cfengine.rb

```diff
@@ -1,8 +1,8 @@
 class Cfengine < Formula
   desc "Help manage and understand IT infrastructure"
   homepage "https://cfengine.com/"
-  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.12.2.tar.gz"
```

Comment:
The package is not available in s3 bucket, reported in upstream repo.

Reply:
> The package is not available in s3 bucket, reported in upstream repo.

The tarball was replaced by the re-released version. So the URL is still just https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.12.2.tar.gz (no `-2`).

Reply:
Thanks for confirming it, I have just updated the sha256 values for the two tarball files.